less/icons.less

```diff
@@ -48,7 +48,8 @@
   &:before {
     content: "\e600";
     //background-color: #FECF03;
-    color: grey;
+    //color: grey;
```

Comment:
Can't really blame you for doing this since it was done previously with the background colors, but generally we should prefer removing old code rather than commenting it out.